package.json

```diff
@@ -5,10 +5,10 @@
   "main": "src/index.js",
   "scripts": {
     "build": "yarn run build:quiet -- --progress",
-    "build:quiet": "yarn run build:clean && NODE_ENV=development webpack",
+    "build:quiet": "yarn run build:clean && env NODE_ENV=development webpack",
```

Comment:
according to https://www.mkssoftware.com/docs/man1/env.1.asp windows has `env` since Vista.